commands/Information/8ball.js

```diff
@@ -0,0 +1,66 @@
+const { Command } = require('klasa');
+const { MessageEmbed } = require('discord.js');
+const fetch = require('node-fetch');
+
+module.exports = class extends Command {
+
+    constructor(...args) {
+        /**
+         * Any default options can be omitted completely.
+         * if all options are default, you can omit the constructor completely
+         */
+        super(...args, {
+            enabled: true,
+            runIn: ['text', 'dm', 'group'],
+            requiredPermissions: [],
+            requiredSettings: [],
+            aliases: [],
+            autoAliases: true,
+            bucket: 1,
+            cooldown: 5,
+            promptLimit: 0,
+            promptTime: 30000,
+            deletable: false,
+            guarded: false,
+            nsfw: false,
+            permissionLevel: 0,
+            description: 'The magic 8ball answers any of your questions. Provided by 8ball.delegator.com',
+            extendedHelp: 'The magic 8ball answers any of your questions. Provided by 8ball.delegator.com',
+            usage: '<question:string>',
+            usageDelim: undefined,
+            quotedStringSupport: false,
+            subcommands: false
+        });
+    }
+
+    async run(message, [question]) {
+        try {
+            let params = encodeURIComponent(question);
+            let uri = 'https://8ball.delegator.com/magic/JSON/' + params;
```

Comment:
Consider storing the 20 possible answers locally. This is better performance wise.
https://en.wikipedia.org/wiki/Magic_8-Ball#Possible_answers